scripts/build_podspecs.sh

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftNIO open source project
+##
+## Copyright (c) 2017-2018 Apple Inc. and the SwiftNIO project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftNIO project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+set -eu
+
+function usage() {
+  echo "$0 [-u] VERSION"
+  echo
+  echo "OPTIONS:"
+  echo "  -u: Additionally, upload the podspecs as they are generated"
+}
+
+upload=false
+while getopts ":u" opt; do
```

Comment:
- I think you don't want the colon in there, right? The colon (usually after an option name) means that the option gets an argument. Or am I missing something.
- Also I'd recommend to do `OPTIND=1` before the `while`. Just in case someone was nasty enough to `export OPTIND=7` or so before running the script